tzatziki-spring-kafka/src/main/java/com/decathlon/tzatziki/steps/KafkaSteps.java

```diff
@@ -304,6 +308,10 @@ public void enable_kafka_interceptor() {
         return publishAvro(name, topic, records);
     }
 
+    private void goBackToBeginningOfTopic(String topic, Consumer<String, ?> consumer) {
```

Comment:
Empty and seems to not be used? I guess it should be deleted?

Reply:
Forgot it 👍